src/modules/fancyzones/FancyZonesLib/FancyZonesData/AppZoneHistory.cpp

```diff
@@ -305,12 +305,12 @@ void AppZoneHistory::AdjustWorkAreaIds(const std::vector<FancyZonesDataTypes::Mo
 {
     bool dirtyFlag = false;
 
-    for (auto iter = m_history.begin(); iter != m_history.end(); ++iter)
```

Comment:
naming is now a bit confusing, since they're now not iters, but direct container values

Reply:
Good point, any suggestion for a name?